watchtower/src/notifier.rs

```diff
@@ -35,12 +43,21 @@ impl Notifier {
             info!("Telegram notifications disabled");
             None
         };
+        let twilio_webhook = if let (Ok(twilio_account), Ok(twilio_token), Ok(twilio_to), Ok(twilio_from)) =
```

Comment:
These four environment variables feel a little clunky, is this a standard for how twilio is used?

If not, maybe adding a single TWILIO_CONFIG along the lines of SOLANA_METRICS_CONFIG would be nicer? 
eg. https://github.com/solana-labs/solana/blob/f4622d67e93cb3f730a618e9e0c708242be700c4/metrics/src/metrics.rs#L365-L390

I don't feel super strongly about this though

Reply:
i can change

Reply:
ok, i made a change, and pushed it... and it's here on the PR... but after that I ran the /ci/test-checks.sh and i think someone else has changed the file as well. But isn't git supposed to prevent that? Honestly not sure what is going on anymore.

Reply:
wait, i think i can fix